drivers/ethernet/eth_w5500.c

```diff
@@ -6,7 +6,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#define DT_DRV_COMPAT	wiznet_w5500
+#define DT_DRV_COMPAT wiznet_w5500
```

Comment:
Could please split the commit?
- format changes
- Macro renames
- API introduction